src/validator.js

```diff
@@ -1,5 +1,17 @@
 import freeze from 'deep-freeze';
```

Comment:
`this.config[field][ruleName]` is used a lot of places in this class. It would improve readability a lot if this was a named function.